### ISSUE TITLE
Question? 

### ISSUE THREAD

**Issue by MrSlipicus** [Viktopia/sonar#60]
Ok, so I have finally after weeks of bashing away at everything gotten an external wifi adapter going, and running in monitor mode. I have gotten Sonar installed and working, I can access it via the browser. 

How long should it be before it starts seeing devices? Our office is chock full of BLE devices, cellphones, even wifi equipment. However, it is doing nothing and showing nothing as far as devices are concerned.

**Comment by vpetersson** [comment]
You should see BLE data within a matter of hours. If not, I'd suggest you poke around in Django admin (and logs) to see what's going on.

With regards to WiFi, that's not supported yet (see https://github.com/databat-io/sonar/issues/10).

**Comment by MrSlipicus** [comment]
So in regards to Django, I did notice an admin login page. However, I never created any credentials? I tried creating them in terminal, but it reports that the Django module cannot be found, is this because it is within the container? Am I doing something wrong, I feel like I am just at the finish line of getting this up and running, as I feel it will be an excellent tool!

**Comment by vpetersson** [comment]
Yeah you need to jump into the container (`docker ps` and `docker exec -ti <container name>`).

**Comment by MrSlipicus** [comment]
Forgive me, I am pretty green to this whole thing. `docker ps` gave me the following result:

`CONTAINER ID   IMAGE            COMMAND                  CREATED          STATUS                         PORTS                                                                          NAMES
15bb42f9e72a   sonar_gunicorn   "/usr/bin/entry.sh /…"   25 minutes ago   Up 24 minutes                  0.0.0.0:80->80/tcp, :::80->80/tcp, 0.0.0.0:8888->8888/tcp, :::8888->8888/tcp   sonar_gunicorn_1
dc0169ca94d9   sonar_celery     "/usr/bin/entry.sh /…"   25 minutes ago   Restarting (1) 8 seconds ago                                                                                  sonar_celery_1
ee145a055053   sonar_redis      "/usr/bin/entry.sh /…"   25 minutes ago   Up 24 minutes                  127.0.0.1:6379->6379/tcp                                                       sonar_redis_1

Which container do I need to be in? 
`

**Comment by vpetersson** [comment]
Should no longer be relevant.
